matter/efr32/efr32mg24/BRD2601B/autogen/sl_component_catalog.h

```diff
@@ -31,15 +31,19 @@
 #define SL_CATALOG_IOSTREAM_PRESENT
 #define SL_CATALOG_MEMORY_MANAGER_PRESENT
 #define SL_CATALOG_MEMORY_PROFILER_API_PRESENT
+#define SL_CATALOG_MPU_PRESENT
```

Comment:
Do we want this? i vaguely remember a discussions were we opted to not add it.

Reply:
Speaking with @jmartinez-silabs we said we _could_ add it. I do not have a strong opinion on this though.